db/postgres/schema/core.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE banks (
+  id INT PRIMARY KEY,
+  bank_name VARCHAR(50) NOT NULL
```

Comment:
```suggestion
  name VARCHAR(50) NOT NULL
```
流儀はあると思うが自分はbankのnameなのでnameでいいように思う